region/norfair/west/Ice Beam Gate Room.json

```diff
@@ -618,15 +618,17 @@
     {
       "id": 28,
       "link": [3, 3],
-      "name": "Sova, Mella, and Sm. Dessgeega Farm",
+      "name": "Mella, and Sm. Dessgeega Farm",
       "requires": [
         {"or": [
```

Comment:
this one could probably also use a reset room, since only the bottom left door would be reasonable to use the farm with